website/forms/php/global-variables/moments.md

```diff
@@ -3,4 +3,19 @@ id: moments
 title: Moments
 ---
 
-*Coming soon*
+### ES_API_KEY_MOMENTS
+
+This variable will set Moments integration api key.
+
+```php
+define('ES_API_KEY_MOMENTS', '<api-key>');
+```
+
+### ES_API_URL_MOMENTS
+
+This variable will set Moments integration api url key.
```

Comment:
```suggestion
Moments API URL.
```

Reply:
I don't like this oversimplification of documentation so I will leave this as is

Reply:
It's dev documentation, should be straight to the point 😄
Setting a constant/variable should imply that something would get overriden.

But if you insist on a less concise variant, can you at least fix the casing on things like "API" and "URL"?

Reply:
Sure but there is no harm in something more human-readable. 

I have updated the api and url case